markdown/org/docs/patterns/waralee/instructions/en.md

```diff
@@ -1,54 +1,73 @@
 
-<Tip>
+<Warning>
 
 ##### The key to Waralee is the waist band
 
 In a way, Waralee is just a waist band with a long piece of fabric attached. The waist band needs to continue from the fabric into two strings of fabric that are used to tie the pants in the front and two in the back. These strings are not included in the pattern. Traditionally, they are of similar length. You can make longer ones for the front panel (the side that is a little shorter than the other, on the left in the illustrations below), to allow you to tie both sets in the front. I prefer this as it is easier to tie, and you don't have the bulk of the knot in the back.
 
-</Tip>
+</Warning>
 
-## Step 1: Sew the pockets
+### Step 1: Sew the pockets
 
-<Note>
+<Fixme>
 
-##### Pockets are optional
-Traditionally, these types of pants do not have pockets. But I want to put my phone, keys and wallet somewhere. So I've included pockets in the pattern. You can decide for yourself if you would like them on your pants.
+The documentation for this step is still being writtern. Sorry for any inconveniences this causes.
 
-The illustrations in the next steps show the Waralee pants without pockets. Only the pocket placements from the pattern are shown, to help you orient yourself.
+</Fixme>
 
-##### If you do want pockets, make them first
-If you're going to make the pockets, make them before putting one piece of fabric on the other. It's easier to work with the parts still separated.
+#### Front Pockets
 
-</Note>
+Okay, the facing is used twice (per pocket). The pocket itself (part 3) is cut-on-fold.
+
+You attach one facing to the pocket after folding the sides under (for finishing), align with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
```

Comment:
```suggestion
You attach one facing to the pocket after folding the sides under (for finishing), aligned with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
```